MarkdownToLatex/MarkdownToLatex/LatexRenderer.cs

```diff
@@ -19,7 +19,7 @@ public static class LatexRenderer
         private static byte oldInList;
 
         /// <summary>a byte indicating whether a list is currently rendered.</summary>
-        public static byte InList {get => _inlist; private set {
+        public static byte InList {get => _inlist; internal set {
```

Comment:
Das was hier gefehlt hat war doch
``` C#
else {
   _inlist = 3;
}
```

,oder?  
Reicht es nicht im Listen Test `        - item` und im Quote `>>>> item` einfügen?

Reply:
Das musste ich manuell machen, weil im Code an sich nie InList / InQuote größer als 3 werden kann. Also im Fall dass InList = 3 ist und man versucht, in die vierte Ebene zu gehen, wird InList gar nicht erst versucht, zu erhöhen.

Reply:
Dann brauchen wir das `if` auch nicht und können es zu einfachen Properties machen + die Felder können dann auch weg.

Reply:
Ok, wird nachgebessert